site/src/ThemeSelector.css

```diff
@@ -44,5 +44,35 @@ svg.icon-layers > path.secondary {
 
 .theme-selector-popover {
   max-height: 80vh;
-  overflow-y: scroll;
+  overflow-y: hidden;
```

Comment:
I believe `overflow-y: auto` will dynamically add/remove the scrollbar at low screen sizes. The scroll bar is clunky, but I would hate to prevent scrolling when needed.

Reply:
Great point Ethan. Yes, my intent was to make sure the theme selector is always navigable. Your suggestion coupled with a parent div with `height: 80vh` or something similar would probably do the tricky without the scrollbars if needed.

Reply:
Good call- scrollbar is now set to 'auto'. 
![image](https://github.com/user-attachments/assets/d2d07a84-15ac-486c-97a4-8f5291ee51d7)